client/components/App.js

```diff
@@ -25,7 +25,7 @@ class App extends Component {
   }
 
   setCurrentItem(item_id){
-    // Updates new item (if user clicks different item, for example)
+    // Updates new item (if user clicks different i4tem, for example)
```

Comment:
is line 28 a typo? is it supposed to be 'i4tem' its a comment so its not a big deal.